src/components/ErrorPage.tsx

```diff
@@ -19,11 +42,9 @@ export const ErrorPage = () => {
         <h1 className="text-4xl mb-8 font-bold" data-testid="unexpected-error">
           An unexpected error occurred
         </h1>
-        {isRouteErrorResponse(error) && (
-          <p className="mb-8">
-            {error.status}: {error.data}
-          </p>
-        )}
+        <p className="mb-8">
```

Comment:
Could you make this `p className="mb-8 w-full overflow-auto">`? On the off chance the message is massive for some reason this will prevent the page from laying out poorly.

Without fix
![image](https://github.com/user-attachments/assets/a37ecf3a-747e-42d2-b522-37abb208d0de)

With fix
![image](https://github.com/user-attachments/assets/20109771-7d84-4c55-8929-75793c683cce)